p2panda-js/src/entry/entry.ts

```diff
@@ -38,8 +38,8 @@ export const signPublishEntry = async (
   const { entryEncoded, entryHash } = signEncodeEntry(
     keyPair,
     operationEncoded,
-    entryArgs.entryHashSkiplink,
-    entryArgs.entryHashBacklink,
+    entryArgs.skiplink,
```

Comment:
Cool, this is also updated in the handbook, but not implemented yet in aquadoggo

Reply:
https://github.com/p2panda/aquadoggo/pull/181

Reply:
In publishEntry it needs to be renamed as well I think 👍

Reply:
Ah, but this is also not reflected here, only in the handbook so far